app/pocket/rpc/v1/openapi.yaml

```diff
@@ -0,0 +1,98 @@
+openapi: 3.0.3
```

Comment:
Thoughts on adding [editor.swagger.io/?url=https://raw.githubusercontent.com/deblasis/pocket/issue/utility_local_proof_of_stake_cli_CLI/app/pocket/rpc/v1/openapi.yaml](https://editor.swagger.io/?url=https://raw.githubusercontent.com/deblasis/pocket/issue/utility_local_proof_of_stake_cli_CLI/app/pocket/rpc/v1/openapi.yaml) to the README?